.github/workflows/command-robotpy-pr.yml

```
@@ -5,7 +5,9 @@ on:
     types:
       - opened
     paths:
-      - 'wpilibNewCommands/**'
+      - 'wpilibNewCommands/**.java'
+      - 'wpilibNewCommands/**.h'
+      - 'wpilibNewCommands/**.cpp'
```

We really only care about changes to the Java version for RobotPy.
```suggestion
      - 'wpilibNewCommands/src/**/*.java'
```